docker/mainnet.env

```diff
@@ -1,4 +1,4 @@
-HEIMDALL_TAG=v0.2.1-mainnet
```

Comment:
We'll be merging to master with this branch name?

Reply:
no for now i have updated it to mumbai-alchemy and mainnet-alchemy